composer.json

```diff
@@ -186,6 +187,9 @@
             "@composer create-test-db",
             "@php -d date.timezone=UTC ./vendor/bin/phpunit -c tests-legacy/old-controllers.xml"
         ],
+        "integration-behaviour-tests": [
+            "@php -d date.timezone=UTC ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml"
```

Comment:
isn't Behavior instead? I'm in doubt here ^^

Reply:
Behavior is the preferred spelling in American English. Behaviour is preferred everywhere else. Both are valid.